source/css/pattern-scaffolding.css

```diff
@@ -252,3 +252,32 @@
 		max-width: 20rem;
 	}
 }
+
+/**
+*  Colors Documentation
+**/
+
+main.colors-documentation h2 {
+	margin-top: 3rem;
+}
+
+main.colors-documentation h3 {
+	margin-top: 2rem;
+}
+
+main.colors-documentation figure {
+	margin: 0;
+	display: grid;
+	grid-auto-columns: 50% 50%;
+	grid: 1fr 1fr;
+	grid-gap: 1rem;
+	grid-auto-flow: column;
+}
+
+main.colors-documentation img {
+	height: auto;
+	max-width: 100%;
+	border-radius: 0.875rem;
+	box-shadow: rgb(60 64 67 / 30%) 0px 1px 2px 0px,
+		rgb(60 64 67 / 15%) 0px 1px 3px 1px;
+}
```

Comment:
this file is provided by pattern lab system itself, so I've introduced a [`pattern-scaffolding-project-specific.scss`](https://github.com/db-ui/core/blob/main/source/css/pattern-scaffolding-project-specific.scss) file within DB UI Core.